implementations/rust/telemetry.md

```diff
@@ -0,0 +1,113 @@
+# Telemetry
+
+Ockam commands and nodes generate telemetry data in the form of Opentelemetry logs and spans.
+This telemetry data is collected via [an Opentelemetry collector](https://opentelemetry.io/docs/collector). That
+collector serves as a central point of collection and can forward this data to a variety of backends, such as:
+
+- S3 for long-term storage.
+- Observability systems like Honeycomb, DataClickHouse, etc... for analysis and visualization.
+
+# Configuration
+
+When the `OCKAM_OPENTELEMETRY_EXPORT` environment variable is set to `true`, there are various ways to send telemetry
+data to the collector:
+
+1. Via a secure channel to your project and then to the collector (the default).
+1. Via a secure channel to your project's authority node and then to the collector.
+1. Via a secure channel to an arbitrary Ockam node and then to the collector.
+1. Directly via HTTP.
+
+## Sending telemetry data directly via a secure channel to the project
+
+This behaviour is controlled by the `OCKAM_TELEMETRY_EXPORT_VIA_PROJECT` environment variable (the default is `true`).
+
+This mode is only active if a default project can be detected locally and is accessible via a secure channel.
+In that case, the telemetry data is sent as Ockam messages to the project node and then forwarded to the collector.
+
+### Services configuration
+
+The project node must be started with the following configuration:
+
+```elixir
+  config :ockam_services,
+    services:
+      {:grpc_forwarder,
+      [
+        address: "grpc_forwarder",
+        grpc_endpoint: "http://opentelemetry-collector:4317",
+        authorization: [{Ockam.Worker.Authorization, :from_secure_channel}]
+      ]}
+```
+
+The value of `grpc_endpoint` must be set to the endpoint of an accessible OpenTelemetry collector.
+
+## Sending telemetry data directly via a secure channel to the authority node
+
+This behaviour is controlled by the `OCKAM_TELEMETRY_EXPORT_VIA_PROJECT` environment variable (the default is `false`).
```

Comment:
Typo in the env var (should point to the authority var right?)

Reply:
Good catch.